### PR TITLE
adding done link to tutorial edit page.  Issue #48

### PR DESCRIPTION
**Pull request by maedae**

### CODE REVIEW

File: app/views/tutorials/view.html.erb [hunk 1]
```diff
@@ -8,6 +8,7 @@
     <%= chapter.description %><br><br>
   <% end %>
   </div>
+
```
Comment (sumeetjain) [inline app/views/tutorials/view.html.erb:11]: Remove trailing whitespace

**Comment by maedae** [comment]
Issue #38 resolved in branch

File: app/controllers/tutorials_controller.rb [hunk 2]
```diff
@@ -57,6 +57,18 @@ def view
...
   end
 
+  def finished_editing_checks
+    @current_user = User.find_by_id(current_user)
+    @tutorial = Tutorial.find_by_id(params[:tutorial_id])
+    draft_check = @tutorial.draft_check
+
+    if draft_check == true
+      redirect_to dashboard_path(session[:company_username])
+    else
```
Comment (sumeetjain) [inline app/controllers/tutorials_controller.rb:67]: Trailing whitespace detected.

File: config/routes.rb [hunk 3]
```diff
@@ -28,6 +28,7 @@
   post 'tutorials/create/confirmation'=> "tutorials#create_confirmation", as: :tutorials_create_confirmation
   get 'tutorials/:tutorial_id/edit' => "tutorials#edit", as: :tutorials_edit
   patch 'tutorials/:tutorial_id/edit/confirmation' => "tutorials#edit_confirmation", as: :tutorials_edit_confirmation
+  get 'tutorials/:tutorial_id/edit/done' => "tutorials#finished_editing_checks", as: :tutorials_finished_editing
```
Comment (sumeetjain) [inline config/routes.rb:31]: Line is too long. [112/80]

**Pull request merged**